### DISCUSSION TITLE
Github copilot accept suggestion line by line | Feature Request

### DISCUSSION THREAD

**Discussion by pawangond** [community/community#20293] [Copilot Conversations]
Currently Copilot in VScode suggest code and we don't always want to take all code but just the 1st line or somepart of it. We need some shortcut to select only somepart of it. Like see the below example. 

Example :
Some shortcut like 
ctrl+shift+[right arrow] takes prediction of one word , press right arrow again (ctrl + shift pressed) takes prediction of next word and so on
ctrl+[rightarrow] takes complete prediction of current selected line
ctrl+shift+[downarrow] takes prediction next line , press downarrow again to select more lines

similarly Up and left arrow to unselect accidently selected
This shortcut keys are just for example , use keys which are not already used by vsccode or something else to avoid conflict.

**Comment by lostlang** [comment]
It is needed on nvim too

**Comment by evrenesat** [comment]
This is definitely a must-have (also for JetBrains IDEs). Most of the time I find myself either ignoring the suggestion or accepting then deleting part of it. I believe the accept/edit case also causes incorrect feedback for the tool, e.g: User accepted my suggestion, I've done it right!

PS: I use PyCharm.

**Comment by bkocis** [comment]
Is there any new on this?

**Comment by zenerbufen** [comment]
VC CODE got this feature. it is needed in other supported IDE's as well!

I'm looking at you VS 2022
